src/ux/wrongNetwork/WrongNetworkComponent.test.tsx

```diff
@@ -88,4 +88,68 @@ describe('Component: WrongNetworkComponent', () => {
       expect(wrapper.find('button')).toHaveLength(0)
     })
   })
+
+  describe('switch to custom network', () => {
+    const rskChains: AddEthereumChainParameter[] = [
+      {
+        chainId: '0x1e',
+        chainName: 'RSK Mainnet',
+        nativeCurrency: {
+          name: 'RSK BTC',
+          symbol: 'RBTC',
+          decimals: 18
+        },
+        rpcUrls: ['https://public-node.rsk.co'],
+        blockExplorerUrls: ['https://explorer.rsk.co'],
+        iconUrls: ['https://developers.rsk.co/assets/img/favicons/android-chrome-192x192.png']
+      },
+      {
+        chainId: '0x1f',
+        chainName: 'RSK Testnet',
+        nativeCurrency: {
+          name: 'Test RSK BTC',
+          symbol: 'tRBTC',
+          decimals: 18
+        },
+        rpcUrls: ['https://public-node.testnet.rsk.co'],
+        blockExplorerUrls: ['https://explorer.testnet.rsk.co'],
+        iconUrls: ['https://developers.rsk.co/assets/img/favicons/android-chrome-192x192.png']
+      },
+      {
+        chainId: '0x4E',
+        chainName: 'RSK Alphanet',
+        nativeCurrency: {
+          name: 'Test RSK BTC',
+          symbol: 'tRBTC',
+          decimals: 18
+        },
+        rpcUrls: ['https://fullnode-use1-1.alphanet.iovlabs.net'],
+        blockExplorerUrls: ['https://explorer.testnet.rsk.co'],
```

Comment:
not sure if this block explorer is right one

Reply:
well, this is just a test. rLogin users are able to provide their own URLs